### ISSUE TITLE
Some PDFs become scrollable when uploaded,

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/vets-design-system-documentation#5196]
`va-file-input` & `va-file-input-multiple` on  `upload` & `upload-supporting-documents`  Some PDFs become scrollable when uploaded, but I cannot click into them, and I don't see to be able to get to them with the keyboard (VO) - Need to confirm with JAWS and NVDA

https://github.com/user-attachments/assets/14f86674-43bb-4508-a89b-1e322962e615
